rpc/js/src/dial.ts

```diff
@@ -218,6 +261,18 @@ export async function dialWebRTC(signalingAddress: string, host: string, opts?:
 	validateDialOptions(opts);
 
 	const webrtcOpts = opts?.webrtcOptions;
+  if (webrtcOpts?.rtcConfig?.iceServers) {
```

Comment:
This should basically follow https://github.com/viamrobotics/goutils/blob/main/rpc/wrtc_client.go#L160-L175 exactly. We always try to extend the list, not when servers are already specified.